docs/.eleventy.js

```diff
@@ -154,7 +161,13 @@ module.exports = function(eleventyConfig) {
 
     eleventyConfig.setLibrary("md",
         markdownIt({ html: true, linkify: true, typographer: true })
-            .use(markdownItAnchor, {})
+            .use(markdownItAnchor, {
+                slugify(text) {
```

Comment:
We can shorten this to just `{ slugify }` using shorthand notation. (Sorry, can’t do multi line suggestions on mobile)

Reply:
NP :) I've updated the PR thanks.